### PR TITLE
description how to terminate o2

### PR DESCRIPTION
**Pull request by justonedev1**

### CODE REVIEW

**Comment by ktf** [comment]
@sawenzel @shahor02 @costing any idea of what is going on with the simulation? Is this a missing object in CCDB?

**Comment by costing** [comment]
The production instance has an object for 310000, but the test one does not. If `ccdb-test.cern.ch` is supposed to be queried for this I can inject the missing fake run number here. But was there a change recently in this logic?

**Comment by sawenzel** [comment]
I don't believe this goes to ccdb-test.cern.ch, also because the CI test itself has been successfully run for the last years. There must be some other reason (network problem)?

File: Framework/Core/README.md [hunk 1]
```diff
@@ -277,6 +277,8 @@ however, no API is provided to explicitly send it. All the created DataChunks ar
 
 When an error happens during processing of some data, the writer of the `process` function should simply throw an exception. By default the exception is caught by the `DataProcessingDevice` and a message is printed (if `std::exeception` derived `what()` method is used, otherwise a generic message is given). Users can provide themselves an error handler by specifying via the `onError` callback specified in `DataProcessorSpec`. This will allow in the future to reinject data into the flow in case of an error.
 
+When the exception is thrown inside processing function it's message and stack trace is printed. However, the application itself is not terminated. If the error encountered is so severe that current workflow cannot continue it is advisable to call `LOG(fatal)` with proper describing message, which makes the application shutdown with non zero error code.
```
Comment (ktf) [inline Framework/Core/README.md]:
```suggestion
When the exception is thrown inside processing function its message and stack trace is printed. However, the application itself is not terminated. If the error encountered is so severe that current workflow cannot continue it is advisable to call `LOG(fatal)` with proper describing message, which makes the application shutdown with non zero error code.
```

**Pull request merged**